README.md

```diff
@@ -1,70 +1,126 @@
+  </ul>
+</details>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+### Key Features <a name="key-features"></a>
 
-### `npm test`
+- SignUp & SignIn
+- View All Cars
+- View Details Of the car
+- Add a car to the page
+- Add a car to Favourites
+- Delete a car
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- **[Generate greeting messages]**
 
-### `npm run build`
+## 🚀 Live Demo <a name="live-demo"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- Front-end deployed website: [Live Demo](car-rental-capstone.netlify.app)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## 💻 Getting Started <a name="getting-started"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+To get a local copy up and running follow these simple example steps.
 
-### `npm run eject`
+### Prerequisites
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+In order to run this project you need:
+- Vistual Studio Code
+- Browser (Firefox / Google Chrome)
+- Node.js (v6.11.2+)
+- linters
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Setup
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Clone this repository to your desired folder:
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+```sh
+  cd my-folder
+  git clone https://github.com/Rhaegar121/Capstone-Front-End.git
+```
 
-## Learn More
+## Install
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+```sh
+ npm install --save -dev
+```
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## Start server
 
-### Code Splitting
+``` npm start ```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## To test the app run following command
 
-### Analyzing the Bundle Size
+``` npm test ```
 
```

Comment:
- [ ] You are instructing your users to run `npm test` to run your tests, however, the only test included is the one that comes with React, and it does not pass:

![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/5a0ad16e-43ba-4444-b0b3-9a57cf60ed71)

![image](https://github.com/Rhaegar121/Capstone-Front-End/assets/57421823/a3af7689-04b8-4fc0-a855-34ff33cda44c)

Kindly remove these test instructions from your readme if you are not including tests for the frontend.

- [ ] Kindly, remove the `App.test.js` that comes with React if not using it.